cmd/network/verification.go

```diff
@@ -280,30 +309,30 @@ func (e *EgressVerification) getSubnetId(ctx context.Context) (string, error) {
 			},
 		})
 		if err != nil {
-			return "", fmt.Errorf("failed to find private subnets for %s: %w", e.cluster.InfraID(), err)
+			return nil, fmt.Errorf("failed to find private subnets for %s: %w", e.cluster.InfraID(), err)
 		}
 
 		if len(resp.Subnets) == 0 {
-			return "", fmt.Errorf("found 0 subnets with kubernetes.io/cluster/%s=owned and %s, consider the --subnet-id flag", e.cluster.InfraID(), e.cluster.InfraID())
+			return nil, fmt.Errorf("found 0 subnets with kubernetes.io/cluster/%s=owned and %s, consider the --subnet-id flag", e.cluster.InfraID(), e.cluster.InfraID())
 		}
 
```

Comment:
```suggestion
	if e.AllSubnets {
	  subnetIds := make([]string, *resp.Subnets)
	  for i, subnet := range *resp.Subnets {
	    subnetIds[i] = subnet.SubnetId
	  }
	  
	  e.log.Info(ctx, "using subnet-ids: %v", subnetIds)
    return subnetIds, nil
	}
```
We can find all the private subnets for non-BYOVPC clusters too